agent/util-scripts/test-bin/rpm

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+args="${@}"
+echo "${0} ${args}" >> ${_testlog}
+
+# We are mocking out how the "rpm" command works by using certain names to
+# drive behaviors.  When not testing require-rpm itself, the special package
+# names are not used, and we assume success and exit with 0.  When the special
+# package names are used, we behave as described for each.
+
+if [[ "${1}" != "--query" ]]; then
+    printf -- "rpm: unexpected 1st argument, '%s'\n" "${1}" >&2
+    exit 101
+fi
+if [[ "${2}" != "--queryformat=%" ]]; then
```

Comment:
I hate to ask this, but does the comparison at line 15 actually work?  I was expecting `{EVR}` after the `%`, and I don't think `!=` does partial matches....

Reply:
I am glad you asked!  That has showed up a hole in our testing, where I did not use `require-rpm` directly with the `bench-scripts`, so we were not testing it properly.

Reply:
Filed as a separate commit with the fix.